### ISSUE TITLE
Notary Project spec and tooling for artifact signing

### ISSUE THREAD

**Issue by SteveLasker** [Azure/acr#417]
Notary v2 is the replacement for Docker Content Trust.
Notary v2 will enable signing of artifacts, which can be copied within or across different registries.
The ability to copy an artifact from one registry (public such as docker hub or mcr) to a private registry is a [best practice](https://stevelasker.blog/2018/11/14/choosing-a-docker-container-registry/). 

However, users must know the content is authentic and signed by the original owner, or signed by additional entities as the artifact moves from public --> dev --> staging --> production --> to archival.

Notary v2 is being developed as a community project: https://github.com/notaryproject/requirements

**Comment by dpaardenkooper** [comment]
Hi ACR product team, 
Any updates about going to GA status?

Kind regards, 
Dinant

**Comment by toddysm** [comment]
@dpaardenkooper Yes, we are wrapping up the security audit for Notary and estimate to complete it by end of the month. Targeting the release of the Notation CLI early to mid May to be able to address any issues that are discovered during the security audit.

**Comment by ezYakaEagle442** [comment]
@toddysm , looking at the doc [https://learn.microsoft.com/en-us/azure/aks/image-integrity?tabs=azure-cli](https://learn.microsoft.com/en-us/azure/aks/image-integrity?tabs=azure-cli) , is Ratify a new incubation project driven my MS or as is it the new name of Notary v2 ?

**Comment by toddysm** [comment]
@ezYakaEagle442 Notary Project and Ratify are two separate OSS projects. Ratify uses the libraries from Notary Project and currently is intended to act as an external data provider for Gatekeeper. Hope this clarifies your question.

Just a note: "Notary V2" is not a term we use anymore because it is not clear what it refers to.

**Comment by yizha1** [comment]
Notary Project's tool Notation support signing artifacts stored in OCI-compliant registries [since 2023](https://notaryproject.dev/blog/2023/announcing-major-release/), see the blog post . The community is working on [Signing arbitrary blobs](https://github.com/notaryproject/notation/issues/741). It is also tracked by issue https://github.com/Azure/acr/issues/801,

`Notary v2` is not a valid term now, Learn more at https://notaryproject.dev/docs/faq/#notary-project-terms

Suggest closing this issue and track the rest of work on issue https://github.com/Azure/acr/issues/801 @toddysm

**Comment by yizha1** [comment]
Closed and Notary Project has already release v1.0.0
